src/azul/terraform.py

```diff
@@ -793,6 +793,17 @@ def tf_config(self, app_name):
         del deployment['lifecycle']
         deployment['triggers'] = {'redeployment': deployment.pop('stage_description')}
 
+        # Instead of specifying this property in the related Terraform
+        # resource, using an AWS API Gateway extension to the OpenAPI
+        # specification lets us avoid maintaining a complicated trigger
+        # dependency between Terraform resources.
+        #
+        swagger_doc = json.loads(locals[app_name])
```

Comment:
```suggestion
        openapi_spec = json.loads(locals[app_name])
```

Above you use "OpenAPI"